### ISSUE TITLE
The program is in progress?

### ISSUE THREAD

**Issue by Comdex** [dpatrie/go-bots#1]
rt.

**Comment by dpatrie** [comment]
Not really. I wanted to build a game like this for a while but always lacking time to do so. This might change pretty soon so stay tuned :).
